core-default/src/main/java/me/aurium/beetle/defaults/task/AbstractTasker.java

```diff
@@ -0,0 +1,43 @@
+package me.aurium.beetle.defaults.task;
+
+import me.aurium.beetle.api.task.SyncQueue;
+import me.aurium.beetle.api.task.TaskRunner;
+import me.aurium.beetle.api.task.Tasker;
+
+import java.util.concurrent.*;
+
+public abstract class AbstractTasker implements Tasker {
+
+    private final SyncQueue syncQueue;
+    private final ExecutorService asyncQueue;
```

Comment:
Is this asyncQueue ever shut down? Any object which contains closable objects or shutdown-able objects should have a close() method itself. I suggest AbstractTasker implement AutoCloseable

Reply:
I can make a wrapper class for the executorservice that implements autoclosable so that tasker, which shouldn't be autoclosable, can remain

Reply:
Hah! It's now a executor, so whoever spawned it can handle it! _disgusting_